### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by igz-gp**
I moved ./pyold/engine to ./rl as I think Gabi and Colleen need it for RL.

### CODE REVIEW

File: index.html
Comment (dfinteligenz) [inline index.html:1]: Is this needed?

File: game-20241114-095044.yaml
Comment (dfinteligenz) [inline game-20241114-095044.yaml:1]: We should remove this

File: analisis-partidas.txt
Comment (dfinteligenz) [inline analisis-partidas.txt:1]: Is this references somewhere, otherwise we should remove this file

File: Makefile
Comment (dfinteligenz) [inline Makefile:1]: I would not remove the Makefile, it's useful for building the engine and generating protobuf and documentation.

Users will not use it as they already have the start-game.sh script, but it's useful for development.

**Review by dfinteligenz — CHANGES_REQUESTED** [review]

File: Makefile
Comment (igz-gp) [reply Makefile:1]: Cool, thanks!

File: analisis-partidas.txt
Comment (igz-gp) [reply analisis-partidas.txt:1]: Yup, already done it! Thanks!

File: index.html
Comment (igz-gp) [reply index.html:1]: No, the Github Pages is configured with jekyll to publish the /front/dist.

File: game-20241114-095044.yaml
Comment (igz-gp) [reply game-20241114-095044.yaml:1]: Yep. Removed!

**Pull request merged**